lib/media/track/localaudiotrack.js

```diff
@@ -35,14 +35,20 @@ class LocalAudioTrack extends LocalMediaAudioTrack {
     super(mediaStreamTrack, options);
 
     const { _log: log } = this;
+    const settings = mediaStreamTrack.getSettings();
 
     Object.defineProperties(this, {
       _currentDefaultDeviceInfo: {
-        value: { deviceId: '', groupId: '', label: '' },
+        value: {
+          deviceId: settings.deviceId,
+          groupId: settings.groupId,
+          label: mediaStreamTrack.label
+        },
```

Comment:
In this case, is there ever a case where `deviceId` or any of the settings come back as undefined? Should that case be handled? 

Something like 
```JavaScript
value: {
    deviceId: typeof settings.deviceId === 'string' ? settings.deviceId : '',
    ...
    }
```